src/views/Schedule2.vue

```diff
@@ -32,9 +33,20 @@ export default Vue.extend({
   data() {
     return {
       ratio: 2.6,
-      width: 195
+      width: 195,
+      nowBarOffset: 0,
+      interval: null
```

Comment:
We may use a more relevant variable name such as nowBarUpdaterInterval what do you think? 🤔

Reply:
Fixed